index.bs

```diff
@@ -261,9 +261,10 @@ spec: SP800-800-63r3; urlPrefix: https://pages.nist.gov/800-63-3/sp800-63-3.html
         text: something you have; url: af
         text: something you are; url: af
 
-spec: webidl; urlPrefix: https://heycam.github.io/webidl
+spec: webidl; urlPrefix: https://webidl.spec.whatwg.org
     type: dfn;
         text: get a copy of the bytes held by the buffer source; url: dfn-get-buffer-source-copy
+        text: Exceptions; url: idl-exceptions
```

Comment:
this shouldn't be needed, fwiw... it should hopefully autolink?

Reply:
Thanks for pointing this out, I'm clearly still learning the ropes. I've replaced this with a simpler `[=simple exceptions=]` as @emlun called out below.